core/src/test/java/com/homeaway/streamplatform/streamregistry/service/ClusterServiceImplTest.java

```diff
@@ -52,24 +50,21 @@
     /**
      * The Infra manager.
      */
-    static InfraManager infraManager;
+    private static InfraManager infraManager;
```

Comment:
why static?
code convention:: If indeed static, please capitalize INFRA_MANAGER

Reply:
This was static before I added the private modifier, but I will capitalize. Whether or not it *ought* to be static I am not entirely sure as it is a test class.